doc/tech/roadmap.md

```diff
@@ -66,5 +66,45 @@ Please refer to individual [release plan](./release_notes/) and features for det
 <a name="3"></a>
 ## Dependencies between various Workstreams
 
+The various work streams in CNTT are 
+- Reference Model (RM)
+- Reference Architecture (RA)
+- Reference Implementation (RI) 
+- Reference Conformance (RC)
+
+The work stream dependency relationship in a simple terms, [Reference Conformance](../ref_cert) verifies and tests the [Reference Implementation](../ref_impl) which follows the requirements and architecture defined in the CNTT [Reference Architecture](../ref_arch) and Reference Architecture describes the high level system components and its interactions by adhering to the requirements and expectations set by the CNTT [Reference Model](../ref_model) which sets the standards for infrastructure abstraction, compliance and verification. 
```

Comment:
typo: "...in a simple terms..." ---> "... in simple terms .."

Reply:
Thanks @pgoyal01. I will correct it.